lib/std/c/darwin.zig

```diff
@@ -1513,3 +1515,178 @@ pub const DB_RECORDTYPE = enum(u32) {
     pub const APP_DEFINED_START = 0x80000000;
     pub const APP_DEFINED_END = 0xffffffff;
 };
+
+pub fn getKernError(err: std.c.kern_return_t) KernE {
+    return @as(KernE, @enumFromInt(@as(u32, @truncate(@as(usize, @intCast(err))))));
+}
+
+pub fn unexpectedKernError(err: KernE) std.posix.UnexpectedError {
+    if (std.posix.unexpected_error_tracing) {
+        std.debug.print("unexpected error: {d}\n", .{@intFromEnum(err)});
+        std.debug.dumpCurrentStackTrace(null);
+    }
+    return error.Unexpected;
+}
```

Comment:
These functions do not belong in the C namespace.

Reply:
Where would you see them? They used to live in `std/c.zig` so that's where code used to import them from (rather through `std.posix.system`).
Eg: https://github.com/mitchellh/libxev/blob/dbe22910a43e9e8ec9948d3cbd73d8488a074967/src/backend/kqueue.zig#L2460

Perhaps in `KERN` ?

Reply:
Yes, I'm sorry. The original contributor put these in the wrong place. I think the best place for such abstractions is in `std.os.darwin`. I can help get this PR landed; I realize it's a bit of a messy situation.